appstore/appstore/settings/base.py

```diff
@@ -235,13 +241,15 @@
 )
 
 # Email configuration
+# UNC Relay: relay.unc.edu:25
+# Renci Relay: relay.renci.org
```

Comment:
I would just remove these kinds of things if we don't need them anymore. If we need to go back and find them later, that's what version control is for